### PR TITLE
Fixed Chapter01 JavaApp

### PR DESCRIPTION
**Pull request by badlogicmanpreet**
Relative path is fixed.

### CODE REVIEW

File: Chapter_01/java-spark-app/src/main/java/JavaApp.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public static void main(String[] args) {
 //           ####### create spark context by starting the cluster locally using 2 CPU cores #######
             sc = new JavaSparkContext("local[2]", "First Spark App");
 
-            JavaRDD<String[]> data = sc.textFile("data/UserPurchaseHistory.csv").map(s -> s.split(","));
+            JavaRDD<String[]> data = sc.textFile("Chapter_01/java-spark-app/data/UserPurchaseHistory.csv").map(s -> s.split(","));
```
Comment (rajdeepd) [inline Chapter_01/java-spark-app/src/main/java/JavaApp.java:20]: Previous path was working fine for me

Comment (badlogicmanpreet) [reply Chapter_01/java-spark-app/src/main/java/JavaApp.java:20]: Will leave as is then, thanks

**Comment by rajdeepd** [comment]
Manpreet do we need to commit this?

**Comment by badlogicmanpreet** [comment]
Not required sir

**Comment by rajdeepd** [comment]
closing as this is not required anymore

**Pull request closed without merging**